presto-docs/src/main/sphinx/optimizer/history-based-optimization.rst

```diff
@@ -31,14 +31,15 @@ The following configuration properties are available for HBO:
 
 ============================================================= =========================================================================================================================== ===================================
 Configuration Property Name                                   Description                                                                                                                 Default value
-===========================================================   =========================================================================================================================== ===================================
+============================================================= =========================================================================================================================== ===================================
```

Comment:
Thanks! The Configuration Properties table builds and is displayed in a local docs build now.